docs/modules/servers/partials/architecture/implemented-standards.adoc

```diff
@@ -37,6 +37,7 @@ This page details standards implemented by the {server-name}.
 - link:https://datatracker.ietf.org/doc/rfc6710/[RFC-6710] SMTP Extension for Message Transfer Priorities
 - link:https://datatracker.ietf.org/doc/rfc8689/[RFC-8689] SMTP Require TLS Option
 - link:https://datatracker.ietf.org/doc/html/rfc1893[RFC-1893] Enhanced Mail System Status Codes
+- link:https://datatracker.ietf.org/doc/html/rfc8461[RFC-8461] SMTP MTA Strict Transport Security (MTA-STS)
```

Comment:
hmm my understanding is that this PR only covers the enforcement part of the MTA-STS rfc as a receiver of emails. 
it does not cover the sending side (RemoteDeliveryMailet) which should : 
- fetch the policy record from DNS
- try to connect to the remote host and start a TLS/SSL session 
- apply the policy (for example refuse to send to hosts that don't have a TLS/SSL capability)

isn't it a bit misleading to state that james implements the standard when the implementation is partial ? (not sure what the track record is here)

Reply:
+1 I will add a section onMTA-STS in the conf